src/components/MovieCard/MovieCard.js

```diff
@@ -1,6 +1,46 @@
 import React from 'react';
 import './MovieCard.scss';
+import PropTypes from 'prop-types';
 
-export const MovieCard = () => (
-  <>Put the card here</>
+export const MovieCard = ({
+  title,
+  description,
+  imgUrl,
+  imdbUrl,
+}) => (
+  <>
+    <div className="card-image">
+      <figure className="image is-4by3">
+        <img src={imgUrl} alt="Film logo" />
+      </figure>
+    </div>
+    <div className="card-content">
+      <div className="media">
+        <div className="media-left">
+          <figure className="image is-48x48">
+            <img src="./images/imdb-logo.jpeg" alt="imdb" />
+          </figure>
+        </div>
+
+        <div className="media-content">
+          <p className="title is-8">{title}</p>
+        </div>
+      </div>
+
+      <div className="content">
+        {description}
+        <a href={imdbUrl}>
+          <br />
+          IMDB
+        </a>
+      </div>
+    </div>
+  </>
 );
+
+MovieCard.propTypes = {
+  title: PropTypes.string.isRequired,
+  description: PropTypes.string.isRequired,
```

Comment:
<img width="551" alt="image" src="https://user-images.githubusercontent.com/38065883/91870299-8c798f80-ec7f-11ea-993a-5b8e734b10a4.png">
make it non required and provide a default value